packages/frontend/src/components/common/Filters/index.tsx

```diff
@@ -50,7 +50,7 @@ const FiltersForm: FC<Props> = ({
 
     const addFieldRule = useCallback(
         (field: FilterableField) => {
-            setFilters(addFilterRule(filters, field));
+            setFilters(addFilterRule({ filters, field }), true);
```

Comment:
I don't think we want to sync the state on add because it won't have any value. So the results would be always empty.